lib/riff/requester.go

```diff
@@ -87,6 +91,19 @@ func (r *Requester) run() {
 	defer Close(r.consumer)
 	defer Close(r.producer)
 
+	client, _, err := zk.Connect([]string{r.zookeeperLocation}, time.Second)
+	if err != nil {
+		log.Errorf("Unable to connect to zookeeper: ", err)
+	}
+	defer client.Close()
+
+	if exists, _, _ := client.Exists("/riffRuns"); !exists {
+		_, err := client.Create("/riffRuns", []byte{}, int32(0), zk.WorldACL(zk.PermAll))
+		if err != nil {
+			log.Fatalf("Unable to create riffRuns node: %v", err)
```

Comment:
log fatal will kill the process. This is probably ok for now (we can't operate without zk working), but imho some retry with logging should be applied later. eventually, we should include things like that in the readiness reported to kubernetes.